tests/api/v1/datadog/api_logs_pipelines_test.go

```diff
@@ -46,15 +46,24 @@ func TestLogsPipelinesLifecycle(t *testing.T) {
 	logMessageRemapper.SetSources([]string{"source"})
 	logMessageRemapper.SetName("log message remapper")
 
-	remapper := datadog.NewLogsAttributeRemapperWithDefaults()
-	remapper.SetSources([]string{"source"})
-	remapper.SetSourceType("tag")
-	remapper.SetTarget("target")
-	remapper.SetTargetType("attribute")
-	remapper.SetPreserveSource(true)
-	remapper.SetOverrideOnConflict(true)
-	remapper.SetName("log message remapper")
-	remapper.SetTargetFormat(datadog.TARGETFORMATTYPE_STRING)
+	remapper1 := datadog.NewLogsAttributeRemapperWithDefaults()
+	remapper1.SetSources([]string{"source"})
+	remapper1.SetSourceType("tag")
+	remapper1.SetTarget("target")
+	remapper1.SetTargetType("attribute")
+	remapper1.SetPreserveSource(true)
+	remapper1.SetOverrideOnConflict(true)
+	remapper1.SetName("log attribute remapper for attribute")
+	remapper1.SetTargetFormat(datadog.TARGETFORMATTYPE_STRING)
+
+	remapper2 := datadog.NewLogsAttributeRemapperWithDefaults()
+	remapper2.SetSources([]string{"source"})
+	remapper2.SetSourceType("tag")
+	remapper2.SetTarget("target")
+	remapper2.SetTargetType("tag")
+	remapper2.SetPreserveSource(true)
+	remapper2.SetOverrideOnConflict(true)
+	remapper2.SetName("log attribute remapper for tag")
```

Comment:
@vinckama can you rename the `remapper1` and `remapper2` variables to provide a reader more context?